### ISSUE TITLE
It seems to stopped working for me today.

### ISSUE THREAD

**Issue by bhowiebkr** [poychang/github-dark-theme#197]
I see there was an update with need tab permissions. Possible that it has not been updated in the chrome store yet?

**Comment by cpercival-decentrix** [comment]
It has stopped working for me too unfortunately. Tried removing and installing again.

**Comment by mkruselj** [comment]
Same here, doesn't work anymore?

**Comment by poychang** [comment]
@bhowiebkr 1.0.54 did not change/add any permissions. The chrome [manifest](https://github.com/poychang/github-dark-theme/commit/4ceab9cd444d64fad8807966050d5cbd99d77e4a#diff-df929923470d7d858d21e28fa8a29c6dL5) only change the version number.

@cpercival-decentrix I just install fresh Chrome and extension, it works fine. And my daily use Edge works fine too.

Can you provide the OS, browser, extension version.

**Comment by bhowiebkr** [comment]
Here's what I've got installed

- GitHub Dark Theme extension - 1.0.54
- Chrome - Version 81.0.4044.122 (Official Build) (64-bit)
- OS: Ubuntu 18.04

**Comment by dbrennan** [comment]
Same here. Windows Chrome Version 83.0.4103.97 (Official Build) (64-bit)
Extension version 1.0.54 as well.
Also not working on Windows Edge Chromium v83.0.478.44 (Official build) (64-bit)

**Comment by poychang** [comment]
How about turn this off

https://github.com/poychang/github-dark-theme/issues/198#issuecomment-639897675

**Comment by dbrennan** [comment]
Yes, it's right in front of my face but I didn't see it. I see it's a new option, and I guess the default of match system theme essentially switches to white if you don't use a dark system theme.

**Comment by mkruselj** [comment]
I forgot to mention. Doesn't work in Waterfox 57 here.

**Comment by mikeweicker** [comment]
> How about turn this off
> 
> [#198 (comment)](https://github.com/poychang/github-dark-theme/issues/198#issuecomment-639897675)

Fixed for me. Not sure how this got turned on.

**Comment by poychang** [comment]
@mkruselj It works with the latest Waterfox.
![image](https://user-images.githubusercontent.com/4879879/84050938-0253bf00-a9e1-11ea-9eda-cb0885f1a595.png)

And do not worry about that option (#198). I remove that part of code in v1.1.1.

**Comment by mkruselj** [comment]
Sorry, I meant Waterfox Classic 57.

**Comment by mkruselj** [comment]
And now Github Dark plugin asked me for new permissions, I granted and IT WORKS!

Thank you! :)

**Comment by justinvforvendetta** [comment]
on chrome, i seemed to fix it by clicking the extensions icon, and then reloading the page. not a great fix but it worked. still love @poychang for making this!
